### ISSUE TITLE
Prepare Monthly JRC for Training

### ISSUE THREAD

**Issue by jamesmcclain** [azavea/noaa-flood-mapping#77]
Consider training a model on a large area using monthly JRC matched to Sentinel-1 scenes and fine-tuning to floods.

To amplify: the motivation here is to take advantage of the fact that the intersection of monthly JRC and Sentinel-1 is so much larger than any of the flood datasets that we have found so far.  Training a model to recognize water on these more plentiful data, then fine-tuning for floods seems like a promising path.

**Comment by moradology** [comment]
After trying and failing to export the global JRC dataset from GEE (the only place where it is currently available), I've found success exporting a subset of the data for the Mississippi river and its environs. This should be sufficient for estimating the efficacy of using JRC as training data. This data lives at s3://global-surface-water/monthly/mississippi-river
